data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/schema/ConsolidationRequest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title" : "ConsolidationRequest",
+  "type" : "object",
+  "required" : [ "source_address", "source_pubkey", "target_pubkey" ],
+  "properties" : {
+    "source_address" : {
+      "type" : "string",
+      "pattern" : "^0x[a-fA-F0-9]{2,}$",
+      "description" : "SSZ hexadecimal",
+      "format" : "bytes"
+    },
```

Comment:
spec says ExecutionAddress is bytes20, can we fix description?

Reply:
Sure. I'll fix it on ExecutionLayerWithdrawalRequest as well =)

Reply:
I think I'll need to look into this as a separate thing because Eth1Address type is mapping to SSZ hexadecimal on the expected description. I won't try to fix it in this PR because it is already big enough.